ruby/reek/default.yml

```diff
@@ -30,7 +30,7 @@ detectors:
     max_calls: 1
     allow_calls: []
   FeatureEnvy:
-    enabled: true
+    enabled: false
```

Comment:
Can you give examples where this metric is not relevant. It is pity to remove this one because it can hide wrong modelling. Can we discuss the pros and cons to remove it.

Reply:
jobteaser/jobteaser#11630

Reply:
In my mind, the problem here is the use of a class with useless state instead of a module that describe better functional programming.
But the interface of middleware does not depend of us.
I think for the moment it is better to disable the metric around the class `:reek: FeatureEnvy`.
It is not because a metric is sometime not relevant that we should remove it every where. Instead, we should take the habit to disable the metric when we are fair with the smell.

Reply:
Another example of a PR with false positive feature envy: https://github.com/jobteaser/jobteaser/pull/11718

Reply:
Also I get false positives when calling an instance method from mapping an instance variable. 

`instance_variable.map { |value| an_instance_method(value) }`

Obviously I see why it raises Feature Envy but for readability's sake this kind of code still makes sense.

Reply:
@AliceLoeser It would be interesting to look into ruby stdlib class to see if there is this kind of code.
The goal of these metrics are also to have code close to ruby style and not rails style.